src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -186,7 +186,7 @@ protected override bool OnShutdown(int timeoutMilliseconds)
             this.shutdownDrainTarget = this.circularBuffer.AddedCount;
             this.shutdownTrigger.Set();
 
-            OpenTelemetrySdkEventSource.Log.DroppedExportProcessorItems(this.droppedCount);
+            OpenTelemetrySdkEventSource.Log.DroppedExportProcessorItems(this.droppedCount, this.GetType());
```

Comment:
Consider moving the `this.GetType()` call into the `if(IsEnabled)` block.

Reply:
True, otherwise it will do unnecessary work when there are 0 dropped items, hopefully the common scenario!